### PR TITLE
Added How to get there

### PR DESCRIPTION
**Pull request by alves-luis**
I originally based this branch off of the video player, therefore I tested it by adding this component underneath the video player. It looks much nicer when it is not right next to the blue-ish component (as it is now).

Any ideas on how to make it look nicer? I think it looks simple and informative.

PS: I just copy-pasted the content from the JOIN website, therefore it is not accurate, it is merely a placeholder

### CODE REVIEW

**Comment by nelsonmestevao** [comment]
I guess we could have a Forum Braga background for this section. What do you think?

File: data/homepage.json [hunk 1]
```diff
@@ -42,5 +42,27 @@
       "Mais novidades serão anunciadas, acompanha-nos nas nossas redes sociais!",
       "Encontramo-nos em breve!"
     ]
+  },
+  "where": {
+    "location": "Altice Fórum Braga",
```
Comment (nelsonmestevao) [inline data/homepage.json]: I think there's no accent in "Forum" :thinking:

File: data/homepage.json [hunk 2]
```diff
@@ -42,5 +42,27 @@
...
+    "subtitle": "Como chegar",
+    "means": {
+      "plane": {
+        "title": "De Avião",
+        "howTo": "O aeroporto mais próximo da cidade de Braga é o Aeroporto Sá Carneiro, no Porto, a cerca de 50 km de distância. A partir deste tem à disposição diversas possibilidades para chegar ao campus de Gualtar: de táxi a viagem demora cerca de 40 minutos e custa, aproximadamente, 60 euros; poderá também utilizar o metro para se deslocar até à estação da CP de Campanhã (linha violeta, Aeroporto - Estádio do Dragão), onde poderá apanhar o comboio suburbano para Braga."
```
Comment (nelsonmestevao) [inline data/homepage.json]: I think I prefer "description" instead of "howTo" :man_shrugging:

**Comment by alves-luis** [comment]
> I guess we could have a Forum Braga background for this section. What do you think?

Great idea! Will look into it!

File: components/homepage/HowToGetHere.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+const SubtitleBox = styled(PaddingBottomBox)({
+  textAlign: 'center',
+  color: 'white',
+});
+
+const GridRow = styled(Grid)({
+  paddingTop: '2em',
+});
+
+function HowToGetHere({ location, subtitle, means }) {
```
Comment (nelsonmestevao) [inline components/homepage/HowToGetHere.js]: Why not an arrow function instead?

File: data/homepage.json (hunk 1 above)
Comment (alves-luis) [reply data/homepage.json]: Apparently it is cool to not write proper portuguese with the naming :astonished: . Will fix it

**Comment by alves-luis** [comment]
Added a background image, to see how it looks like. The thing is, the text obviously cannot stay white. Maybe adding a rectangular box with some transparency behind each wall of text would make it look clearer? But then we wouldn't be able to see the background that well :confused:

**Comment by nelsonmestevao** [comment]
I think we could add the rectangular box you mention and trying to add some effects to the image itself, dimming the light or some other filter.

I think it is better with the image but the filter should help keeping the site more consistent :muscle:

**Comment by miguelbrandao** [comment]
It's looking really good, but I also agree with @nelsonmestevao . Maybe a semitransparent [Paper component](https://material-ui.com/components/paper/) behing each one would look better?

**Comment by alves-luis** [comment]
> It's looking really good, but I also agree with @nelsonmestevao . Maybe a semitransparent [Paper component](https://material-ui.com/components/paper/) behing each one would look better?

What do you think?

File: components/homepage/HowToGetHere.js [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+        icon={faPlane}
+        desc={means.plane.description}
+      />
+      <Mean name={means.bus.title} icon={faBus} desc={means.bus.description} />
+      <Mean
+        name={means.train.title}
+        icon={faTrain}
+        desc={means.train.description}
+      />
+      <Mean name={means.car.title} icon={faCar} desc={means.car.description} />
```
Comment (miguelbrandao) [inline components/homepage/HowToGetHere.js]: I think this should be a map, so that we can add and remove means on a whim

**Pull request merged**